salt/pillar/git_pillar.py

```diff
@@ -365,12 +365,9 @@
 from salt.exceptions import FileserverConfigError
 from salt.pillar import Pillar
 
-# Import third party libs
-from salt.ext import six
-
-PER_REMOTE_OVERRIDES = ('env', 'root', 'ssl_verify', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'mountpoint', 'all_saltenvs')
-GLOBAL_ONLY = ('base', 'branch')
+PER_REMOTE_OVERRIDES = ('base', 'env', 'root', 'ssl_verify', 'refspecs', 'fallback', 'saltenv_whitelist', 'saltenv_blacklist', 'disable_saltenv_mapping', 'ref_types,)
```

Comment:
`'fallback'` isn't yet available in 2019, this cause an error loading git_pillar

> 2020-06-25 11:05:45,429 [salt.utils.gitfs :2226][CRITICAL][13627] Key 'git_pillar_fallback' not present in global configuration. This is a bug, please report it.
> 2020-06-25 11:05:45,430 [salt.master      :642 ][CRITICAL][13627] Failed to load git_pillar
> 2020-06-25 11:05:45,430 [salt.master      :643 ][CRITICAL][13627] Master failed pre flight checks, exiting

solved when removed